client/src/components/utils/index.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
why does eslint error happen? 
@ranasobeid95

Reply:
we need many components to be exported, so I use Named Export here nested of default